src/scripts/openai-api.js

```diff
@@ -0,0 +1,33 @@
+import OpenAi from 'https://cdn.jsdelivr.net/npm/openai@4.22.1/+esm'
+
+
+
+async function generateResponseApi(actor, input) {
+    const openai = new OpenAi({
+        apiKey: await actor.getFlag("unkenny", "llmAPIKey"),
```

Comment:
I think we need some error handling when fetching flags, even though it bloats the code. Otherwise, actors created with previous versions fail rather silently.

Reply:
Did some copy paste... Shall we create a new method which adds error handling to getFlag?

Reply:
Kept as it is, since codebase is mimimal and parameters are subject to change.